invenio_rdm_records/services/schemas/metadata.py

```diff
@@ -32,13 +32,17 @@ def _valid_url(error_msg):
     return validate.URL(error=error_msg)
 
 
-class AffiliationSchema(Schema):
+class VocabularySchema(Schema):
+    """Invenio Vocabulary schema."""
+
+    id = SanitizedUnicode(required=True)
+    title = fields.Dict(dump_only=True)
+
+
+class AffiliationSchema(VocabularySchema):
     """Affiliation of a creator/contributor."""
 
-    name = SanitizedUnicode(required=True)
-    identifiers = IdentifierSet(
-        fields.Nested(IdentifierSchema),
-    )
+    name = SanitizedUnicode()
```

Comment:
Yeah... this is becoming tricky in the frontend. I'd say that in general most of them will match `name == title.en`. However, for example a French intitution the "original" name might be `title.fr` (e.g. CERN, though I'm not sure about this anymore). However, is good to have the titles indexed because people will have hits for suggestions no matter the language the write in. On the other hand, they might be a bit puzzled of why it appears back in another language. e.g.

- Search for: nucleaire
- Get back: European Center for Nuclear Research (i.e. nuclear instead of nucleaire)